project/src/com/google/daggerquery/protobuf/autogen/BindingGraphProto.java

```diff
@@ -0,0 +1,1932 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
All these files with `Proto` suffix are generated automatically, so don't waste your time on them ⏰

Reply:
How about excluding generated code from source control, by marking them in `.gitignore`? And maybe add a command for the code generation in the `README`. This way it will reduce clutter and prevent the generated / actual source code from ever going out of sync :)

Reply:
Cool, thanks! I think it's good idea, I will add entire `autogen` folder unto .gitignore